src/main/java/karuna/karuna_backend/Authentication/JWT/JwtConfig.java

```diff
@@ -0,0 +1,26 @@
+package karuna.karuna_backend.Authentication.JWT;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+
+@Component
+@Getter
+@Setter
+@NoArgsConstructor
+@ConfigurationProperties(prefix = "application.jwt")
+public class JwtConfig {
+    private String secretKey ;
+    private String tokenPrefix;
+    private Duration tokenExpirationTime;
+    private Duration refreshTokenExpirationTime;
+
+    public String getAuthorizationHeader() {
```

Comment:
its needed?

Reply:
refreshTokenExpiration not now, but I've built setup for next issue that will be provided about jwt